src/relay/backend/executor.cc

```diff
@@ -91,7 +91,8 @@ TVM_REGISTER_EXECUTOR("aot")
     .add_attr_option<Bool>("link-params", Bool(true))
     .add_attr_option<Bool>("unpacked-api")
     .add_attr_option<String>("interface-api")
-    .add_attr_option<Integer>("workspace-byte-alignment");
+    .add_attr_option<Integer>("workspace-alignment")
```

Comment:
i still kind of like having a unit here...just curious what's the motivation for dropping it?

Reply:
i think we still need the unit put back here

Reply:
reverted back to `workspace-byte-alignment` and for `constant-byte-alignment`